### ISSUE TITLE
Unable to see Pictures at Blogpost near 'Last Activity [x]h ago'

### ISSUE THREAD

**Issue by bonsaibauer** [lexiconhq/lexicon#25]
### Describe the bug
It seems that the Profile Pictures are intact, so 'top Participants' at Post and in Profil you can see a Picture, but not in front of the Postowner.

### *REQUIRED:* Include the contents of `api/.env` and `frontend/.env`
_If you are manually overriding any environment variables when running the package scripts, include those as well._

**api/.env**
```
PROSE_DISCOURSE_HOST=https://discourse.fridaysforfuture.me
```

**frontend/.env**
```
MOBILE_PROSE_HOST=https://apid.fridaysforfuture.me
```

### _REQUIRED:_ answer the following questions:
- If this is related to your mobile app:
  - Are you running it through Expo, or did you build it?
  -> Running through Expo
  - Is it running on your mobile device or on a simulator?
  -> Mobile device

### To Reproduce
1. Go to 'npx expo start'
2. Scan QR Code

### Screenshots
```
@dev:/var/lexicon/frontend# npx expo start
Starting project at /var/lexicon/frontend
Starting Metro Bundler
› Metro waiting on exp://128.140.13.149:19000
› Scan the QR code above with Expo Go (Android) or the Camera app (iOS)

› Press a │ open Android
› Press w │ open web

› Press j │ open debugger
› Press r │ reload app
› Press m │ toggle menu

› Press ? │ show all commands

Logs for your project will appear below. Press Ctrl+C to exit.
iOS Bundling complete 85032ms
 ERROR  Warning: Cannot update a component (`PostProvider`) while rendering a different component (`Home`). To locate the bad setState() call inside `Home`, follow the stack trace as described in https://reactjs.org/link/setstate-in-render
    in Home (created by SceneView)
    in StaticContainer
    in EnsureSingleNavigator (created by SceneView)
    in SceneView (created by BottomTabView)
    in RCTView (created by View)
    in View (created by Screen)
    in RCTView (created by View)
    in View (created by Background)
    in Background (created by Screen)
    in Screen (created by BottomTabView)
    in RNSScreen (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by InnerScreen)
    in Suspender (created by Freeze)
    in Suspense (created by Freeze)
    in Freeze (created by DelayedFreeze)
    in DelayedFreeze (created by InnerScreen)
    in InnerScreen (created by Screen)
    in Screen (created by MaybeScreen)
    in MaybeScreen (created by BottomTabView)
    in RNSScreenNavigationContainer (created by ScreenContainer)
    in ScreenContainer (created by MaybeScreenContainer)
    in MaybeScreenContainer (created by BottomTabView)
    in RCTView (created by View)
    in View (created by SafeAreaInsetsContext)
    in SafeAreaProviderCompat (created by BottomTabView)
    in BottomTabView (created by BottomTabNavigator)
    in Unknown (created by BottomTabNavigator)
    in BottomTabNavigator (created by TabNavigator)
    in TabNavigator (created by SceneView)
    in StaticContainer
    in EnsureSingleNavigator (created by SceneView)
    in SceneView (created by CardContainer)
    in RCTView (created by View)
    in View (created by CardContainer)
    in RCTView (created by View)
    in View (created by CardContainer)
    in RCTView (created by View)
    in View
    in CardSheet (created by Card)
    in RCTView (created by View)
    in View (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by PanGestureHandler)
    in PanGestureHandler (created by PanGestureHandler)
    in PanGestureHandler (created by Card)
    in RCTView (created by View)
    in View (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by Card)
    in RCTView (created by View)
    in View (created by Card)
    in Card (created by CardContainer)
    in CardContainer (created by CardStack)
    in RNSScreen (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by InnerScreen)
    in Suspender (created by Freeze)
    in Suspense (created by Freeze)
    in Freeze (created by DelayedFreeze)
    in DelayedFreeze (created by InnerScreen)
    in InnerScreen (created by Screen)
    in Screen (created by MaybeScreen)
    in MaybeScreen (created by CardStack)
    in RNSScreenContainer (created by ScreenContainer)
    in ScreenContainer (created by MaybeScreenContainer)
    in MaybeScreenContainer (created by CardStack)
    in RCTView (created by View)
    in View (created by Background)
    in Background (created by CardStack)
    in CardStack (created by HeaderShownContext)
    in RCTView (created by View)
    in View (created by SafeAreaInsetsContext)
    in SafeAreaProviderCompat (created by StackView)
    in RCTView (created by View)
    in View (created by GestureHandlerRootView)
    in GestureHandlerRootView (created by StackView)
    in StackView (created by StackNavigator)
    in Unknown (created by StackNavigator)
    in StackNavigator (created by StackNavigator)
    in StackNavigator (created by SceneView)
    in StaticContainer
    in EnsureSingleNavigator (created by SceneView)
    in SceneView (created by CardContainer)
    in RCTView (created by View)
    in View (created by CardContainer)
    in RCTView (created by View)
    in View (created by CardContainer)
    in RCTView (created by View)
    in View
    in CardSheet (created by Card)
    in RCTView (created by View)
    in View (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by PanGestureHandler)
    in PanGestureHandler (created by PanGestureHandler)
    in PanGestureHandler (created by Card)
    in RCTView (created by View)
    in View (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by Card)
    in RCTView (created by View)
    in View (created by Card)
    in Card (created by CardContainer)
    in CardContainer (created by CardStack)
    in RNSScreen (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by InnerScreen)
    in Suspender (created by Freeze)
    in Suspense (created by Freeze)
    in Freeze (created by DelayedFreeze)
    in DelayedFreeze (created by InnerScreen)
    in InnerScreen (created by Screen)
    in Screen (created by MaybeScreen)
    in MaybeScreen (created by CardStack)
    in RNSScreenContainer (created by ScreenContainer)
    in ScreenContainer (created by MaybeScreenContainer)
    in MaybeScreenContainer (created by CardStack)
    in RCTView (created by View)
    in View (created by Background)
    in Background (created by CardStack)
    in CardStack (created by HeaderShownContext)
    in RCTView (created by View)
    in View (created by SafeAreaInsetsContext)
    in SafeAreaProviderCompat (created by StackView)
    in RCTView (created by View)
    in View (created by GestureHandlerRootView)
    in GestureHandlerRootView (created by StackView)
    in StackView (created by StackNavigator)
    in Unknown (created by StackNavigator)
    in StackNavigator (created by RootStackNavigator)
    in RootStackNavigator (created by AppNavigator)
    in EnsureSingleNavigator
    in BaseNavigationContainer
    in ThemeProvider
    in NavigationContainerInner (created by AppNavigator)
    in AppNavigator (created by App)
    in UserEventProvider (created by App)
    in ModalProvider (created by App)
    in PostProvider (created by App)
    in ThemeProvider (created by App)
    in AppearanceProvider (created by App)
    in RNCSafeAreaProvider (created by SafeAreaProvider)
    in SafeAreaProvider (created by App)
    in StorageProvider (created by App)
    in ApolloProvider (created by App)
    in App (created by withDevTools(App))
    in withDevTools(App)
    in RCTView (created by View)
    in View (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer
    in main(RootComponent)

LOG  VirtualizedList: You have a large list that is slow to update - make sure your renderItem function renders components that follow React performance best practices like PureComponent, shouldComponentUpdate, etc. {"contentLength": 4172.5, "dt": 2368, "prevDt": 2702}
```

#### Desktop (please complete the following information):
-

#### Smartphone (please complete the following information):
 - Device: iPhone 11
 - OS: iOS16.3.1
- Expo Go App IOS

#### GraphQL API (please complete the following information):
Please indicate where the Prose GraphQL API is running.

Are you only running it locally on your development machine?
-> https://apid.fridaysforfuture.me

Have you deployed it somewhere? If so, what domain or IP address did you deploy it at?
-> https://apid.fridaysforfuture.me

How specifically did you configure the mobile app to connect to the API?
- I didn't configure the mobile App only .env in /api and /frontend

### Additional context
Another Issue, when iam clicking on the Post, the App crashed...

**Comment by bonsaibauer** [comment]
![image](https://user-images.githubusercontent.com/129884416/230982589-d658627c-3913-4f02-b945-25a334d6be9f.png)

**Comment by Kinostrome** [comment]
We did attempt to reproduce this, although we were unable to verify with your particular site since it requires approval to register. However, we suspect this issue may no longer be occurring in the newer release.

The team will be off for the next week or so for a national holiday. But after the break, we're looking to get an alpha branch up on this repo that hopefully does not contain this issue, as well as a few others (such as crashing due to certain `upload://` urls).

**Comment by bonsaibauer** [comment]
I thank you for your work and time on the project. Do you think it has something to do with private discourse or were you also able to reproduce the error on a public forum?

 I will try it out to narrow down the issue. Thank you and have a great national holiday next week!

**Comment by jovitazhang** [comment]
Hi @bonsaibauer hope you're doing great!
We wanted to follow up regarding the issue you reported. 
We kindly request that you update to the latest stable version v2, and thoroughly test the application. If you encounter any further issues or have any questions, please don't hesitate to reach out to us. We're here to assist you.

In the meantime, we'll consider this ticket closed, assuming that the issue has been resolved. However, if you do experience any problems or need further assistance, please don't hesitate to reopen this ticket, and we'll promptly investigate and address any concerns you may have.

Thank you for your patience and understanding. We value your feedback and are committed to providing the best possible experience with our product.
